services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/CustomerWithContacts.kt

```diff
@@ -77,12 +77,13 @@ data class Contact(
 data class Customer(
     val type: CustomerType?, // Mandatory in Allu, but not in drafts.
     val name: String,
-    val country: String, // ISO 3166-1 alpha-2 country code
+    val postalAddress: PostalAddress?,
     val email: String?,
     val phone: String?,
     val registryKey: String?, // y-tunnus
     val ovt: String?, // e-invoice identifier (ovt-tunnus)
     val invoicingOperator: String?, // e-invoicing operator code
+    val country: String, // ISO 3166-1 alpha-2 country code
```

Comment:
Couldn't country have stayed where it was?

Reply:
Now it is removed completely as it is not needed in Haitaton.